lib/sequelize.js

```diff
@@ -952,7 +953,7 @@ class Sequelize {
    * const cls = require('continuation-local-storage');
    * const ns = cls.createNamespace('....');
    * const Sequelize = require('sequelize');
-   * Sequelize.cls = ns;
+   * Sequelize.useCls(ns);
```

Comment:
Yes. New API. Where before you would instruct Sequelize to use a CLS namespace with `Sequelize.cls = ns`, now it's `Sequelize.useCls(ns)`.

Two reasons for this:

1. Necessity: `cls-bluebird` requires the CLS namespace to be provided at the time it applies the patch. So the patch can't be applied statically. The user needs to set the CLS namespace and *then* the patch is applied.

2. Advantage: the cls-bluebird patch is only applied to `Sequelize.Promise` when the user calls `.useCls()`. So for the majority of people who don't use CLS, they avoid the overhead of the patch which should improve performance.